src/docfx/Models/BuildJsonConfig.cs

```diff
@@ -123,5 +123,8 @@ public class BuildJsonConfig
 
         [JsonProperty("customLinkResolver")]
         public string CustomLinkResolver { get; set; }
+
+        [JsonProperty("versions")]
+        public Dictionary<string, VersionInfo> Versions { get; set; }
```

Comment:
is it possible to merge them into one centralized place? the FileMappingItem also has dest, looks confusing for me..

---
In reply to: [102859173](https://github.com/dotnet/docfx/pull/1342#discussion_r102859173) [](ancestors = 102859173,102691611)